### PR TITLE
[Doc Issue][API-based ME manifest attribute update]

### PR DESCRIPTION
**Pull request by v-ypalikila**
updated manifest and properties from commands and composeExtension to commands and composeExtensions.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/688800bf4ba650033988258f18e8478c7b480466/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/156b19e8-3546-4d5a-9139-e810acc7eda9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/688800bf4ba650033988258f18e8478c7b480466/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 93 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dde42461-3219-421c-91ef-40d975bccd8f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/064a473c5c9603af8c92856a625b51d639a659ed/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c4a29a8c-96d8-46d2-a678-3787afe29ea8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/064a473c5c9603af8c92856a625b51d639a659ed/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 93 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4a93ae1f-799f-40d3-8ab0-8546094fc7c9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b92c3d61c922a2d445589c42e17d7ebe47571dd7/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0610979-6da1-4c84-b2c1-4d295fe99e1b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b92c3d61c922a2d445589c42e17d7ebe47571dd7/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 92 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7eb0bccb-97dd-4df3-9663-99ed734b27a4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0ceeb1b5a9576ab10413ce19aa201449d5641b80/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b437dab1-cab0-4f51-9233-8c00d23810b6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0ceeb1b5a9576ab10413ce19aa201449d5641b80/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 92 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/621b382d-8a31-46cd-bd80-bc1d387bd96a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/17fe4a7e66ccc3e2e7d47693f6467e2aa124db6c/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a727272e-0be9-43f9-aec7-ac34c220c0a3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/17fe4a7e66ccc3e2e7d47693f6467e2aa124db6c/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 93 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aa842727-42cb-4e15-bcc3-176f57d4756a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/898c9ccc46d463b5d2a15982d2bbbe5765d2045e/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7be81987-f529-4c2e-af6f-a457045b3af3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/898c9ccc46d463b5d2a15982d2bbbe5765d2045e/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 94 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8ba9a6fc-7bb8-42e9-bb23-9e21367f9078) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9adec1c3e8fbb2be5b569ab839112a6485949fc5/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5369dccd-1f66-4cc2-b6bf-2122673a6d1e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9adec1c3e8fbb2be5b569ab839112a6485949fc5/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 94 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d01b2259-ffef-45d2-8847-474d14904edf) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7cdea81e16d504b1f1504bd3aa73887843257bb6/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fb826210-9702-426d-8632-18781b2094ef) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7cdea81e16d504b1f1504bd3aa73887843257bb6/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 94 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/519b22b0-dfc2-4160-afdb-7b8c4d8e0577) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a815c8025d870a0e935098402c5bc6bc9e57cc10/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/991420bb-f78e-4c9e-a3c6-db08ccf1bf95) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a815c8025d870a0e935098402c5bc6bc9e57cc10/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 94 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a4c81d78-30de-4f19-a0b8-b85d2daf21c6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/798b21f206184fd79896e5fe9cefad0c9b2233cc/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/96cfb3b3-d25f-4379-9bb8-0dc8a279bbb5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/798b21f206184fd79896e5fe9cefad0c9b2233cc/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 94 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f0acd0a8-926f-4e15-9eac-ec63c6bbf315) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/798b21f206184fd79896e5fe9cefad0c9b2233cc/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Fdefine-search-command.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/551da31d-9301-4969-921e-4b02431213a7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/api-based-overview.md [hunk 1]
```diff
@@ -382,68 +382,74 @@ Update app manifest (previously called Teams app manifest) with the `composeExte
...
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "accentColor": "#FFFFFF",
+  "composeExtensions": [
+    {
+      "composeExtensionType": "apiBased",
+      "apiSpecificationFile": "openapi.yml",
```
Comment (ChetanSharma-msft) [inline msteams-platform/messaging-extensions/api-based-overview.md:412]: This looks wrong:
.yaml not .yml

Also verify the file name as well.

File: msteams-platform/messaging-extensions/api-based-overview.md [hunk 2]
```diff
@@ -382,68 +382,74 @@ Update app manifest (previously called Teams app manifest) with the `composeExte
...
-              "parameters": [
-                {
-                  "title": "Filter",
-                  "name": "filter",
-                  "description": "Filter repairs by who they're assigned to."
-                }
-              ],
-              "apiResponseRenderingTemplateFile": "listrepairsresponsetemplate.json"
+              "name": "countryCode",
+              "title": "Country Code",
```
Comment (ChetanSharma-msft) [inline msteams-platform/messaging-extensions/api-based-overview.md:426]: Make sure these parameters are added as per the Yaml file configuration.

File: msteams-platform/messaging-extensions/api-based-overview.md [hunk 3]
```diff
@@ -382,68 +382,74 @@ Update app manifest (previously called Teams app manifest) with the `composeExte
...
+          "title": "API for fetching Klarna."
         }
-      ],
-      "validDomains": [
-        "repairs-api-2023.azurewebsites.net"
       ]
+    }
+  ],
+  "validDomains": [
+    "klarna.com/us/shopping"
```
Comment (ChetanSharma-msft) [inline msteams-platform/messaging-extensions/api-based-overview.md:437]: Check if this domain is correct.

File: msteams-platform/messaging-extensions/api-based-overview.md [hunk 4]
```diff
@@ -382,68 +382,74 @@ Update app manifest (previously called Teams app manifest) with the `composeExte
...
+  ],
+  "webApplicationInfo": {
+    "id": "228a4704-1cbb-4e59-b026-3b194021dd86"
```
Comment (ChetanSharma-msft) [inline msteams-platform/messaging-extensions/api-based-overview.md:440]: What is this id?

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8c17be201dd7210e5eccbe7c0fcdd035d959e04e/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/880155c2-2eca-41c6-8087-0fab85c98971) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8c17be201dd7210e5eccbe7c0fcdd035d959e04e/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 94 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3f14d369-e0ea-41b3-a225-329658c1eb66) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8c17be201dd7210e5eccbe7c0fcdd035d959e04e/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Fdefine-search-command.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1ec8a071-58d7-4112-a885-91306a3e0ac9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/api-based-overview.md (hunk 4 above)
Comment (v-ypalikila) [reply msteams-platform/messaging-extensions/api-based-overview.md:440]: updated the manifest matching the sample with AI Tools and OpenAPI Description Document.

File: msteams-platform/messaging-extensions/api-based-overview.md (hunk 3 above)
Comment (v-ypalikila) [reply msteams-platform/messaging-extensions/api-based-overview.md:437]: updated the manifest matching the sample with AI Tools and OpenAPI Description Document.

File: msteams-platform/messaging-extensions/api-based-overview.md (hunk 2 above)
Comment (v-ypalikila) [reply msteams-platform/messaging-extensions/api-based-overview.md:426]: updated the manifest matching the sample with AI Tools and OpenAPI Description Document.

File: msteams-platform/messaging-extensions/api-based-overview.md (hunk 1 above)
Comment (v-ypalikila) [reply msteams-platform/messaging-extensions/api-based-overview.md:412]: @ChetanSharma-msft  All the YAML files we use and also the format in the manifest is .yml and not YAML.

Sharing a screenshot of the manifest with .yml we tested and got the output.
image

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9225d54d855687728966115d12672b3f7ac29be8/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8c43b0e1-e3fd-4168-8dfd-291baea915f4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9225d54d855687728966115d12672b3f7ac29be8/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 94 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2c5e9175-4c76-48fc-bbaf-7ac1ea485ebe) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9225d54d855687728966115d12672b3f7ac29be8/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Fdefine-search-command.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/08a11977-28ed-4df7-b08d-04781d876989) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/538bc47dea356fad4bd19107e658b1bcb1ba030d/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0ae3b364-9b00-4e51-9227-f765a30744d9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/538bc47dea356fad4bd19107e658b1bcb1ba030d/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 94 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9bf888f3-f85b-4d86-a1d8-6d2334cd802d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/538bc47dea356fad4bd19107e658b1bcb1ba030d/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Fdefine-search-command.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fff72f76-6910-47d1-a276-e33dc7ab6e61) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/api-based-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/586de29fb57b813060dab92af5ddcb936fa50b3c/msteams-platform%2Fmessaging-extensions%2Fapi-based-overview.md) | 90 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/01b6ed71-b310-47e5-a49c-2cc6d3b26fdd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/586de29fb57b813060dab92af5ddcb936fa50b3c/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 94 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4f8a69cc-7520-4944-89b9-d883ef292c54) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/586de29fb57b813060dab92af5ddcb936fa50b3c/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Fdefine-search-command.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3578ed2d-bd61-4181-92f3-5745cd5f4fd3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**